final_prep/mod_zero_hero.rb

```diff
@@ -1,41 +1,61 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
-
+hero_name = "Garbage Gal"
+special_ability = "turn people into garbage"
+puts "My name is #{hero_name}."
+puts "I have the power to #{special_ability}."
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
-#   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "Have no fear! #{hero_name} is here!"
 
+#   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+catchphrase = "WHA-PA! I have the power to #{special_ability}!"
 # Declare two variables - power AND energy - set to integers
-
+power = 5
+energy = 10
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
-#   full_energy should add 150 to your current energy
+full_power = power * 500
 
+#   full_energy should add 150 to your current energy
+full_energy = energy + 150
 # Declare two variables - is_human and identity_concealed - assigned to booleans
+is_human = false
+identity_concealed = false
 
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
+arch_enemies = ["General Alt-Right","Anti-Vax Anne", "Jeffery Bezos"]
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+sidekicks = ["Compost Kyle", "Daniel Tiger", "Recycle Reese"]
 # Print the first sidekick to your terminal
-
+p sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+p arch_enemies[2]
```

Comment:
Nice use of bracket notation. A next level to this could be to find a way to print the last element no matter how many elements are in the array.